pkg/volume/flexvolume/api/objects.go

```diff
@@ -0,0 +1,72 @@
+package api
+
+import (
+	"k8s.io/kubernetes/pkg/api"
+)
+
+// Probe plugin
+
+type FlexVolumeDriverCapabilities struct {
+	AccessMode          api.PersistentVolumeAccessMode
+	DynamicProvisioning bool
+	Attachment          bool
+	LocalAttachment     bool // When Attachement is true, issue the attach/detach calls from the target host, not the controller.
+	CustomMount         bool // This driver has a custom mount/unmount logic. Mount/Unmount must be called.
```

Comment:
What does CustomMount mean ? It is not mentioned in the original proposal.
Also, does this field make sense at all, if Attachment is true. As far as the current proposal goes, if Attachment is true, then mount will not get called .....

(I am another enthusiastic user of FlexVolume, and very interested in changes in this area. Hopefully I will be able to help in contributing soon).